auth/service.py

```diff
@@ -134,4 +139,22 @@ def get_user_profile(self, token: str) -> User:
       user = self.repository.find_user_by_email(decoded['email'])
       if not user:
         raise UserNotFound
-      return user
\ No newline at end of file
+      return user
+    
+    def get_all_users_basic_info(self, token: str) -> User:
+      self.logger.debug("jwt token", token)
+      if token == "" or token == None:
```

Comment:
I think you can refactor this auth process

Reply:
Okey, I will refactor those auth process 😄